source/libraries/ContractDeployer.ts

```diff
@@ -291,11 +292,11 @@ export class ContractDeployer {
         return market;
     }
 
-    private async createBinaryMarket(universe, endTime: number, feePerEthInWei: number, denominationToken, designatedReporterAddress, numTicks: number): Promise<ContractBlockchainData> {
+    private async createBinaryMarket(universe, endTime: number, feePerEthInWei: number, denominationToken, designatedReporterAddress, numTicks: number): Promise<{ [methodName: string]: ContractMethod }> {
```

Comment:
We should probably create a new type for `Contract` that this returns (as well as updating `parseAbiIntoMethods` to return it instead of its inline type).